src/quera_ahs_utils/plotting.py

```diff
@@ -164,15 +164,19 @@ def plot_avg_density(densities, register, with_labels = True, custom_axes = None
 
             register (AtomArrangement): The register used in creating the Hamiltonian.
 
-            with_labels (Boolean): Choose if each atom's index is displayed over the atom itself in the resulting figure. 
+            with_labels (Boolean, optional): Choose if each atom's index is displayed over the atom itself in the resulting figure. 
```

Comment:
```suggestion
            with_labels (Boolean): Default is True. Choose if each atom's index is displayed over the atom itself in the resulting figure. 
```
I'd prefer keeping the exact type the argument is supposed to have in the parentheses with nothing else (I worry `optional` could be misinterpreted as *another* type). It might be best to just put the "Default is..." line right after the colon to make it more clear in all applicable cases.